File: 0485-max-consecutive-ones/0485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let current = 0;
    let max = 0;
    for(let i = 0;i<nums.length;i++){
        if (nums[i] === 1){
             current += 1
        }else {
            max = Math.max(current,max)
            current = 0
        }
    }
    max = Math.max(current,max)
    return max
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int result = 0, count = 0 ;
        for(int i = 0 ; i < nums.Length ; i++){
            if(nums[i] == 1){
                count++;
                result = Math.Max(result, count);
            }else{
                count = 0 ;
            }
        }
        return result;
    }
}